nextflow.config

```diff
@@ -134,7 +135,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = true
-    validationSchemaIgnoreParams     = 'cf_ploidy,genomes,igenomes_base'
+    validationSchemaIgnoreParams     = 'input_restart,cf_ploidy,genomes,igenomes_base,ascat_alleles,ascat_genome,ascat_loci,ascat_loci_gc,ascat_loci_rt,bwa,bwamem2,cf_chrom_len,chr_dir,dbsnp,dbsnp_tbi,dbsnp_vqsr,dict,dragmap,fasta,fasta_fai,germline_resource,germline_resource_tbi,intervals,known_indels,known_indels_tbi,known_indels_vqsr,known_snps,known_snps_tbi,known_snps_vqsr,mappability,ngscheckmate_bed,pon,pon_tbi,sentieon_dnascope_model,snpeff_db,snpeff_genome,vep_cache_version,vep_genome,vep_species'
```

Comment:
Seems strange that `validationSchemaIgnoreParams` should ignore so many parameters? 🤔  Is that just some kind of temporary fix?